packages/scripts/config/webpack.config.js

```diff
@@ -121,6 +126,11 @@ if ( ! isProduction ) {
 	// WP_DEVTOOL global variable controls how source maps are generated.
 	// See: https://webpack.js.org/configuration/devtool/#devtool.
 	config.devtool = process.env.WP_DEVTOOL || 'source-map';
+	config.module.rules.unshift( {
```

Comment:
It might be `push` as well. I don't know if order matters. Better safe than sorry :)